### DISCUSSION TITLE
HW 4 Part 3 Q 3

### DISCUSSION THREAD

**Discussion by caroline-cook** [cooperrc/computational-mechanics#58] [Q&A]
I was wondering if anyone knew how to go about solving for the y intercept and slope for the USA and EUR plots? I know that's how I can figure out what the energy use will be in 2025, but I'm not sure how to build that equation to find out. Thanks!

**Comment by cooperrc** [comment]
There are 2 approaches:
a, b = np.polyfit(year, energy, 1)​ -> now the energy use is $a\cdot year + b​$
A = np.polyfit(year, energy, 1)​ -> now the energy use is `np.polyval(A, year)`
